packages/calculator/src/dragablecalculator.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import Calculator from '@pie-framework/material-ui-calculator';
+import { withStyles } from 'material-ui/styles';
+import Draggable from 'react-draggable';
+
+const styles = {
```

Comment:
please use 2 spaces for indentation - setting up an `.editorconfig` file shortly